mkdocs/cli.py

```diff
@@ -120,13 +122,14 @@ def json_command(clean, config_file, strict, site_dir):
 @cli.command(name="gh-deploy")
 @click.option('--clean', is_flag=True, help=clean_help)
 @click.option('--config-file', type=click.File('rb'), help=config_file_help)
-def gh_deploy_command(config_file, clean):
+@click.option('--commit-message', is_flag=False, help=commit_message_help)
```

Comment:
Pull requests welcome to add short names :)

--message does sound like the best choice. Let's use that. You can add the short version of you like, otherwise it can be added with all the others later.